src/encoded/static/components/navigation/components/UserActionDropdownMenu.js

```diff
@@ -20,6 +20,30 @@ import { LoginNavItem, LogoutDropdownItem } from './LoginNavItem';
  * @property {boolean|function} active - Whether action is currently active.
  */
 
+
+ const FFAuth0Options = {
+     auth: {
+         sso: false,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 8 spaces but found 9. (indent)](https://app.codacy.com/app/4dn/fourfront/pullRequest?prid=4484611)